doc/tutorials/add_test_case.md

```diff
@@ -0,0 +1,337 @@
+# Add support for a PTS profile
+
+- [Plan the BTP interface](#plan-the-btp-interface)
+- [Get familiar with tools](#get-familiar-with-tools)
+- [Add a new profile](#add-a-new-profile)
+- [Add a new test case](#add-a-new-test-case)
+- [Synchronize multiple Lower Testers](#synchronize-multiple-lower-testers)
+- [Keep global variables in Stack](#keep-global-variables-in-stack)
+
+## Plan the BTP interface
+
+PTS opened in automation mode sends MMI requests to AutoPTS who handles them

```

Comment:
PTS **running** in automation mode sends MMI requests to AutoPTS **which** handles them